bower.json

```diff
@@ -7,8 +7,7 @@
     "ember-qunit-notifications": "0.1.0",
     "Faker": "~3.0.0",
     "pretender": "~0.10.1",
-    "lodash": "~3.7.0",
-    "bootstrap-sass": "^3.3.6"
+    "lodash": "~3.7.0"
```

Comment:
Seems like maybe accidentally removed during merge. Can you please add this back?